service/expression.js

```diff
@@ -25,8 +26,13 @@ class ExpressionService {
   }
 
   async removeExpression(expressionId) {
-    await expressionRepository.deleteExpression(expressionId);
-    await expressionRepository.deleteExpressionFromUsers(expressionId);
+    const deleted = await expressionRepository.deleteExpression(expressionId);
+
+    if (!deleted) {
+      throw new InexistentItem('ExpressionId does not exists.');
+    }
+
+    expressionRepository.deleteExpressionFromUsers(expressionId);
```

Comment:
I might be wrong but shouldn't you await this or return it to be awaited in the controller?